### PR TITLE
feat: navigation to group conversation details page [AR-1715]

### PR DESCRIPTION
**Pull request by saleniuk**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1715" title="AR-1715" target="_blank">StoryAR-1715</a>  Navigation to group conversation details page (Skeleton)
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

We need to be able to show group conversation details and in order to do that, we need to navigate from conversation screen.

### Solutions

Added new NavigationItem, updated ConversationScreen top bar to make the whole title clickable instead of just the dropdown icon, created base group conversation details screen with proper top bar and two tabs.

#### How to Test

Click on group converation title dropdown to navigate to the details.

### Attachments (Optional)

device-2022-06-20-171556 (1)

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-684/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-684/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3677430)(3677430)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3677430)(3677430)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt [hunk 1]
```diff
@@ -201,6 +203,19 @@ enum class NavigationItem(
         }
     },
 
+    GroupConversationDetails(
```
[the remark this answers is not in the document]
Comment (saleniuk) [reply app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt:206]: @MohamadJaara actually I thought about that, but according to the design specification:
> In 1:1 conversations, the conversation  details are not available. Instead, the user profile of the other user is shown.
Further design specs of the user profile view can be found here: https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/?node-id=0%3A17597

so there is no "conversation details" screen for 1:1 conversations 😅

**Review by MohamadJaara — APPROVED** [review]
lgtm 🚀

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-684/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3683586)(3683586)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3683586)(3683586)-dev-release.apk)

**Pull request merged**